Felienne/hedy
# [UI IMPROVEMENT] Improves programs user overview

**Description**
In this PR we improve the programs overview of a user to clearly differentiate between adventures. Currently the adventure name is printed directly in the program name, which is unclear. We fix this by separately showing the adventure name, if any is there.

**Fixes**
This PR fixes #2509

**How to test**
Navigate to your programs page and verify that the "program box" has a new layout. Also verify that the adventure name is correctly shown (translated) on the corresponding column. When no adventure name is found, "-" is shown.

![Schermafbeelding 2022-05-02 om 08 53 19](https://user-images.githubusercontent.com/38155424/166195965-d05a4a2c-ba53-4408-acb0-0db7a1521e7f.png)
